gae/js/app/views/staffposition.js

```diff
@@ -0,0 +1,126 @@
+        'app/views/staffposition_rates',
+        'text!app/templates/staffposition.html',
+    ],
+    function(Backbone, Backform, RatesView, template) {
+
+        var StaffPositionFlow = Backbone.View.extend({
+            el: '#simple-form-view',
+            events: {
+                'click button[name=saveStaffPosition]': 'saveStaffPosition',
+            },
+            initialize: function(options) {
+                var self = this;
+                this.options = options;
+                self.staffposition = options.staffposition;
+                self.template = _.template(options.template);
+                self.loading = options.loading;
+                self.hourly_rates = self.staffposition.get('hourly_rates');
+                self.mileage_rates = self.staffposition.get('mileage_rates');
+                self.hourlyView = options.hourlyView;
+                self.mileageView = options.mileageView;
+
+                 this.listenTo(this.staffposition, 'change',
+                    function(staffposition){
+                        if (self.loading){
+                            self.loading = false;
+                            self.hourly_rates = self.staffposition.get('hourly_rates');
+                            self.mileage_rates = self.staffposition.get('mileage_rates');
+                            self.makeForm(self.staffposition, self.options.fields).render();
+                        }
+                    });
+                 this.makeForm(this.staffposition, this.options.fields).render();
+                 this.on('change-rates', function(){this.render(true);});
+            },
+            makeForm: function(mdl, fields){
+                this.form =  new Backform.Form({
+                    model: mdl,
+                    fields: fields,
+                    showRequiredAsAsterisk: true,
+                });
+                return this;
+            },
+            render: function(display_alert) {
+                if (!this.loading){
+                    this.$el.html(this.template({display_alert: display_alert}));
+                    this.hourlyView.setElement('#hourly-rates').render();
+                    this.mileageView.setElement('#mileage-rates').render();
+                    this.form.setElement('#staffposition-form').render();
+                    this.form.$el.find('label.control-label:contains("*")').addClass('required');
+                    this.form.$el.find('input').first().focus();
+
+                    if(display_alert){
+                        $('#staffposition-form > div.form-group > div > button')
+                            .html('Save all changes including unsaved rate changes');
+                    }
+                }
+            },
+            saveStaffPosition: function(e){
+                var self = this;
+                $(e.target).prop('disabled', true);
+                this.staffposition.set({
+                    'position_name': this.form.model.get('position_name'),
```

Comment:
The way I read makeForm and its call points, this.staffposition and this.form.model are the same object.  So this block doesn't do anything useful.